extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -624,19 +624,24 @@ export class AmpA4A extends AMP.BaseElement {
           }
           if (fetchResponse.headers && fetchResponse.headers.has(
               EXPERIMENT_FEATURE_HEADER_NAME)) {
-            fetchResponse.headers.get(EXPERIMENT_FEATURE_HEADER_NAME).split(',')
-              .forEach(pair => {
-                const parts = pair.split('=');
-                if (parts.length != 2 || !parts[0]) {
-                  dev().warn(TAG, `invalid experiment feature ${pair}`);
-                  return;
-                }
-                this.postAdResponseExperimentFeatures[parts[0]] = parts[1];
-              });
-            this.cspEnabled_ =
-              this.postAdResponseExperimentFeatures[CSP_ENABLED_EXP_NAME] ==
-                'true';
+            this.populatePostAdResponseExperimentFeatures_(
+                fetchResponse.headers.get(EXPERIMENT_FEATURE_HEADER_NAME));
+          }
+          if (getMode().localDev && this.win.location &&
+              this.win.location.search) {
+            // Allow for setting experiment features via query param which
+            // will potentially override values returned in response.
+            const match = /(?:\?|&)a4a_feat_exp=([^&]+)/.exec(
```

Comment:
this doesn't do anything to protect against the value of the param being something bad except & but I don't think that matters for local dev anyways

Reply:
right, this is local dev only and the value is still consumed by individual features.